universe/Species.cpp

```diff
@@ -146,6 +146,24 @@ std::string Species::Dump() const {
     return retval;
 }
 
+std::string Species::GameplayDescription() const
+{
+    std::string result(UserString(m_gameplay_description));
+
+    for (std::vector<boost::shared_ptr<Effect::EffectsGroup> >::const_iterator it = m_effects.begin();
+         m_effects.end() != it; ++it) {
+        if (it == m_effects.begin())
+            result += "\n";
+
+        std::string description = (*it)->GetDescription();
```

Comment:
If GetDescription is modified to return a const std::string& then so can the type of this variable that stores it.